### ISSUE TITLE
A few questions regarding the training and inference setup.

### ISSUE THREAD

**Issue by sadasdadda** [haoyuhan1/RoE#2]
Hello, thank you for your impressive work on RoE.
 I have a few questions regarding the training and inference setup.
1.The training hyperparameters (learning rate, batch size, optimizer, number of epochs)
2.The hardware configuration (GPU type and number)
3.The approximate training time and inference latency
Any additional insights for successful reproduction would be highly appreciated.
Thank you for your time and support!
